homeassistant/components/cover/device_action.py

```diff
@@ -139,6 +140,8 @@ async def async_call_action_from_config(
     elif config[CONF_TYPE] == "set_tilt_position":
         service = SERVICE_SET_COVER_TILT_POSITION
         service_data[ATTR_TILT_POSITION] = config["position"]
+    elif config[CONF_TYPE] == "stop_tilt":
+        service = SERVICE_STOP_COVER_TILT
```

Comment:
We should not make changes to the the core component and to MQTT if the focus of the PR is to add a feature to MQTT cover